contracts/AtlasMineStakerUpgradeable.sol

```diff
@@ -185,11 +185,10 @@ contract AtlasMineStakerUpgradeable is
      * @param _amount               The amount of tokens to deposit.
      */
     function deposit(uint256 _amount) public virtual override nonReentrant {
+        require(false, "deposits currently paused");
         require(!schedulePaused, "new staking paused");
         require(_amount > 0, "Deposit amount 0");
 
-        _updateRewards();
-
```

Comment:
not needed i think since `require(false)` always false

(but i get if you want to remove for clarity)